cilium/datadog_checks/cilium/data/conf.yaml.example

```diff
@@ -0,0 +1,34 @@
+init_config:
+
+instances:
+
+    ## @param agent_endpoint - string - required
+    ## The URL where your application metrics are exposed by Prometheus.
+    ## By default, the Cilium integration collects `cilium-agent` metrics.
+    #
+  - agent_endpoint: http://localhost:9090/metrics
+
+    ## @param operator_endpoint - string - optional
+    ## If you would like to collect the `cilium-operator` metrics, enable
+    ## the `collect_operator_metrics` option.
+    ## Cilium operator metrics are exposed on port 6942.
+    #
+    # operator_endpoint: http://localhost:6942/metrics
+
+    ## @param collect_operator_metrics - boolean - optional - default: false
+    ## Enable to collect `cilium-operator` metrics.
+    #
+    # collect_operator_metrics: false
+
+    ## @param tags - list of key:value elements - optional
+    ## List of tags to attach to every metrics, events and service checks emitted by this integration.
+    ## Learn more about tagging: https://docs.datadoghq.com/tagging/
+    #
+    # tags:
+    #   - <KEY_1>:<VALUE_1>
+    #   - <KEY_2>:<VALUE_2>
+
+    ## @param prometheus_timeout - integer - optional - default: 10
+    ## Overrides the default timeout value in second
+    #
+    # prometheus_timeout: 10
```

Comment:
Why not just `timeout`?

Reply:
The openmetrics config currently uses `prometheus_timeout`. https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py#L237-L239

We haven't implemented the requests wrapper in OpenmetricsBase yet to handle remapping. But I think we could use `timeout` config and include in the openmetrics instance like
`'prometheus_timeout': instance.get('timeout', 10)`.